snippets/logging-elastic-dep-snip.adoc

```diff
@@ -0,0 +1,12 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+//
+:_content-type: SNIPPET
+
+[NOTE]
+====
+ As of logging version 5.4.3 the Elasticsearch Operator is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to using the Elasticsearch Operator to manage the default log storage, you can use the Loki Operator.
```

Comment:
Might want to reframe it with the exact release we are planning to sunset ES support in Logging.

Reply:
This is the required version of the verbiage, we don't specify an exact release until we're 100% on it.